packages/shared/lib/services/plans/plans.ts

```diff
@@ -51,6 +51,21 @@ export async function updatePlan(db: Knex, { id, ...data }: Pick<DBPlan, 'id'> &
     }
 }
 
+export async function updatePlanByTeam(
+    db: Knex,
+    { account_id, ...data }: Pick<DBPlan, 'account_id'> & Partial<Omit<DBPlan, 'id' | 'account_id'>>
```

Comment:
is `account_id` required in the `Omit` since it is in `Pick`?

Reply:
yes, it translates to `{ account_id: string } & EverythingElseIsOptional<>`
I need the pick to avoid having account_id being set has optional too